system-parachains/bridge-hubs/bridge-hub-polkadot/src/weights/snowbridge_pallet_ethereum_client.rs

```diff
@@ -111,7 +111,7 @@ impl<T: frame_system::Config> snowbridge_pallet_ethereum_client::WeightInfo for
 		//  Measured:  `92787`
 		//  Estimated: `93857`
 		// Minimum execution time: 169_086_936_000 picoseconds.
-		Weight::from_parts(169_539_621_000, 0)
+		Weight::from_parts(69_539_621_000, 0)
```

Comment:
It's a small tweak we did with @bkontur to check that snowbridge tests were passing aside the overweight factor that was happening. After (the bench bot is fixed 😅 and) weights are refreshed, we'll have to check why this call is causing to exceed max block weight.

Reply:
@pandres95 I did some digging and this weight is correct. The test that is failing has more extrinsic than it used to, so it fails trying all the extrinsics in one block. It wasn't caught on Westend because Westend's block max block weights are higher. I will fix the test. :)

Reply:
@pandres95 fix over here: https://github.com/pandres95/runtimes/pull/3

Reply:
Yes. I was kinda suspicious that though the weight might be good, there was something smelly about the overall execution of the extrinsics.

Reply:
Yes. I was kinda suspicious that though the weight might be good, there was something smelly about the overall execution of the extrinsics.

Thanks a lot for the fix 😁